tracer/test/test-applications/security/Samples.Security.AspNetCore5/Controllers/HomeController.cs

```diff
@@ -5,6 +5,7 @@
 using System.Diagnostics;
 using System.Linq;
 
+#pragma warning disable ASP0019 // warning ASP0019: Use IHeaderDictionary.Append or the indexer to append or set headers. IDictionary.Add will throw an ArgumentException when attempting to add a duplicate key
```

Comment:
Any reason to not change the actual `Response.Headers.Add()` below to use the indexer, instead of ignoring the warning?

Reply:
No idea, up to ASM IMO 🤷‍♂️ 😄